proxies/live/apiproxy/policies/AssignMessage.Errors.CatchAllTarget.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Errors.CatchAllTarget">
+    <Set>
+      <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
+      {
+        "resourceType": "OperationOutcome",
+        "meta": {
```

Comment:
Needs meta.lastUpdated. Should be current date time. Speak to Jack as he has to add for rate limiting.

Reply:
Added current_timestamp as per Jacks changes